Container Services/creating-a-docker-swarm-in-clc.md

```diff
@@ -0,0 +1,175 @@
+{{{
+  "title": "Creating a Docker Swarm in CLC",
+  "date": "07-09-2016",
+  "author": "Richard Case",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+![logo](../images/docker.png)
+
+The release of Docker Engine v1.12 introduced swarm mode for creating and managing a cluster of Docker Engines which is referred to as a swarm. This replaces Docker Swarm which was previously a separate application.
+
+A Swarm is a collection (cluster) of nodes running Docker Engine following a decentralized design where services can be deployed. Swarm mode includes scaling, service discovery, desired state and many other features. For an overview of Swarm mode see: https://docs.docker.com/engine/swarm/
+
+By using our plugin for Docker Machine, customers can create a Docker swarm on CenturyLink Cloud by running a small number of Docker Machine commands to create the nodes. We can then run the Docker CLI commands required to create the swarm by setting the machine context using Docker Machine.
+
+## Before We Start
```

Comment:
@richardcase in case you didn't see this you need to use ### for section headers .... this won't be merged until that is updated